### PR TITLE
Getting ready for 2023

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
@starbelly, it's a big change, so it might be best to just pull it, run the Jekyll stuff and see how it looks. I'll try to give some pointers in self-review mode for you to pay more attention to. Thanks.

**Note**: most of the changes in img/, legacy/ and vendor/ are just uninteresting.

**Note**: I also made most of the links to images and whatnot complete, so it's easier to spot where they're coming from, sometimes we had to assume /img, sometimes we assumed /img/legacy, and this was confusing.

**Note**: I'm going back on the Ruby gems' versions, since they appear to not be supported by Netlify 🤷.

### CODE REVIEW

File: .gitignore
Comment (paulo-ferraz-oliveira) [inline .gitignore:1]: Being more thorough

File: .tool-versions
Comment (paulo-ferraz-oliveira) [inline .tool-versions:1]: Version bumps only

File: Gemfile
Comment (paulo-ferraz-oliveira) [inline Gemfile:1]: Version bumps only

File: Gemfile.lock
Comment (paulo-ferraz-oliveira) [inline Gemfile.lock:1]: Auto-generated

File: README.md
Comment (paulo-ferraz-oliveira) [inline README.md:1]: See if these work for you...

File: _layouts/default.html
Comment (paulo-ferraz-oliveira) [inline _layouts/default.html:1]: Some names changed, or files were moved, but the content is basically the same. I just wanted to be more organised.

File: _config.yml [hunk 1]
```diff
@@ -14,207 +12,151 @@ nav:
...
-    twitter: expede
-    image: judges_2022/Brooklyn.png
-  - name: Quinn Wilton
-    twitter: wilton_quinn
-    image: judges_2022/Quinn.jpg
+  #- name: name
+  #  twitter: twitter
+  #  image: /img/judges_2023/image
 
 rules:
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:118]: Rule changes start here. They are basically tweaks to existing rules.

**Comment by starbelly** [comment]
@paulo-ferraz-oliveira We should update the logo in this PR as well. It's in our slack channel.

**Comment by paulo-ferraz-oliveira** [comment]
Logo updated. Resolve the conversations if you're Ok with everything I write :)

**Comment by starbelly** [comment]
> @paulo-ferraz-oliveira We should update the logo in this PR as well. It's in our slack channel.

Right, so only issue right now is the logo. Some padding was removed such that the logo ends up under the main nav menu.

File: css/style.css [hunk 2]
```diff
@@ -486,14 +503,9 @@ header.masthead .img-container .header-logo {
...
-.header-logo-small-top {
-    width: 25%;
-    margin-top: 3rem;
-}
-
 .header-logo-small-left {
-    width: 25%;
-    margin-bottom: 3rem;
+    width: 375px;
+    margin-top: 4rem;
```
Comment (starbelly) [inline css/style.css:508]:
```suggestion
    margin-top: 5rem;
    margin-bottom: 1rem;
```

**Review by starbelly — APPROVED** [review]
💪

**Pull request merged**